interface/reports/encounters_report.php

```diff
@@ -53,8 +54,8 @@ function show_doc_total($lastdocname, $doc_encounters) {
   }
 }
 
-$form_from_date = fixDate($_POST['form_from_date'], date($DateFormat));
-$form_to_date = fixDate($_POST['form_to_date'], date($DateFormat));
+$form_from_date = fixDate($_POST['form_from_date'], date('Y-m-d'));
```

Comment:
is there a problem with the configurable date format here that we need a static Y-m-d?